Portal/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,11 @@
-﻿<!DOCTYPE html>
+﻿@{
+    var requestCultureInfo = Context.Features.Get<Microsoft.AspNetCore.Localization.IRequestCultureFeature>()?.RequestCulture.Culture;
+    var isEnglish = Constants.CultureConstants.English.Name == requestCultureInfo?.Name;
+    var oppositeCultureValue = isEnglish ? Constants.CultureConstants.Welsh.Name : Constants.CultureConstants.English.Name;
+    var oppositeCultureName = isEnglish ? "Welsh" : "English";
```

Comment:
I would put this in some sort of helper class, and inject it into the page

Reply:
Done.